### PR TITLE
Add Github PR template

### PR DESCRIPTION
**Pull request by alexanderbez**
Introduce a Github PR template based off of the Cosmos SDK template.

### CODE REVIEW

**Comment by alexanderbez** [comment]
@jackzampolin, we should also add an issue template and CODEOWNERS file as well. Wanna tackle this?

**Comment by jackzampolin** [comment]
I'll take the issue template and `CODEOWNERS`. Thanks for grabbing the PR template!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺ 
+v                               ✰  Thanks for creating a PR! ✰    
+v    Before smashing the submit button please review the checkboxes. 
+v    If a checkbox is n/a - please still include it but + a little note why
+☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  --> 
+
+* [ ] Updated all relevant documentation in docs
+* [ ] Updated all code comments where relevant
+* [ ] Wrote tests
+* [ ] Updated CHANGELOG.md
```
Comment (jackzampolin) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe we leave out the `CHANGELOG` till we have the first release?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+* [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))
+* [ ] Added appropriate labels to PR (ex. wip, ready-for-review, docs)
```
Comment (jackzampolin) [inline .github/PULL_REQUEST_TEMPLATE.md]: This can't be done by non-members so maybe we pull this off? I'm happy to triage issues so we can avoid it

**Review by jackzampolin — COMMENTED** [review]
Just a couple of small points

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (alexanderbez) [reply .github/PULL_REQUEST_TEMPLATE.md]: Sure thing

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (alexanderbez) [reply .github/PULL_REQUEST_TEMPLATE.md]: Mhmm tbh, I think we should leave it as a team member may be able to edit the description and add the appropriate labels. I dont wanna pin all the work on you ;-)

**Comment by alexanderbez** [comment]
Updated @jackzampolin

**Comment by jackzampolin** [comment]
Bomb! LGTM!

**Pull request merged**